src/app/about-app/about-app/about-app.component.html

```diff
@@ -87,6 +87,9 @@ <h2 class="header-3 my-4">{{ "PARTICIPANTS" | translate }}</h2>
             <li>
               <b>Kancelaria Prezesa Rady Ministrów</b>
             </li>
+            <li>
+              <b>Snowflake</b>
+            </li>
             <li>
               <b>Tech for the Rescue</b>
```

Comment:
to się chyba nazywa Tech To The Rescue
https://www.techtotherescue.org/

Reply:
potwierdzilem nazwe i poprawilem